src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -191,17 +205,18 @@ public static AbstractBuilding createFromNBT(Colony colony, NBTTagCompound compo
             try
             {
                 building.readFromNBT(compound);
-            }
-            catch (RuntimeException ex)
+            } catch (RuntimeException ex)
             {
-                Log.logger.error(String.format("A Building %s(%s) has thrown an exception during loading, its state cannot be restored. Report this to the mod author",
-                                        compound.getString(TAG_BUILDING_TYPE), oclass.getName()), ex);
+                Log.logger.error(String
+                        .format("A Building %s(%s) has thrown an exception during loading, its state cannot be restored. Report this to the mod author", compound
+                                .getString(TAG_BUILDING_TYPE), oclass.getName()), ex);
                 building = null;
             }
-        }
-        else
+        } else
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)